src/components/ButtonPanel.js

```diff
@@ -1,43 +1,46 @@
-/* eslint-disable no-unused-vars */
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
-import '../styles/styles.css';
 
-const ButtonPanel = () => {
+const ButtonPanel = ({ clickHandler }) => {
   const Gray = '#e0e0e0';
   return (
     <div className="buttonpanel">
       <div>
-        <Button name="AC" color="{Gray}" />
-        <Button name="+/-" color="{Gray}" />
-        <Button name="%" color="{Gray}" />
-        <Button name="÷" />
+        <Button key="AC" name="AC" color={Gray} clickHandler={clickHandler} />
+        <Button key="+/-" name="+/-" color={Gray} clickHandler={clickHandler} />
+        <Button key="%" name="%" color={Gray} clickHandler={clickHandler} />
+        <Button key="÷" name="÷" clickHandler={clickHandler} />
       </div>
       <div>
-        <Button name="7" color="{Gray}" />
-        <Button name="8" color="{Gray}" />
-        <Button name="9" color="{Gray}" />
-        <Button name="X" />
+        <Button key="7" name="7" color={Gray} clickHandler={clickHandler} />
+        <Button key="8" name="8" color={Gray} clickHandler={clickHandler} />
+        <Button key="9" name="9" color={Gray} clickHandler={clickHandler} />
+        <Button key="X" name="X" clickHandler={clickHandler} />
       </div>
```

Comment:
The multiplication is not working, i guess it's because you put `'X'` here...meanwhile, your case statements was expecting `'*'` (a star)